LICENSE

```diff
@@ -1,12 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 Khan Academy
-
-This software also uses portions of the underscore.js project, which is
-MIT licensed with the following copyright:
-
-Copyright (c) 2009-2015 Jeremy Ashkenas, DocumentCloud and Investigative
-Reporters & Editors
+Copyright (c) 2013-2018 Khan Academy
```

Comment:
This looks right to me, but perhaps someone from KA can confirm.

Also, is it best practice to call the file `LICENSE` or `LICENSE.txt`? I thought the latter but I don't know.

Reply:
@edemaine Most JavaScript library repos I know use `LICENSE` and GitHub initializes the repo with `LICENSE`.